### ISSUE TITLE
Active Nav component links override accessible name

### ISSUE THREAD

**Issue by smhigley** [OfficeDev/office-ui-fabric-react#11788]
### Environment Information

- **Package version(s)**: latest / 7.84.1
- **Browser and OS versions**: all

### Describe the issue:

The `selectedAriaLabel` property adds an `aria-label` with the defined text to the currently selected link. This means that the selected link will have its accessible name overridden by generic "selected" text, making it inaccessible. Even if all links have individual `ariaLabel` properties and the "selected" text is appended to that `ariaLabel`, including the active state of a link in its accessible name is not good practice.

Ideally the active link should have `aria-current="page"` set, and the accessible name should not be modified.

### Reproduction:
https://fabricweb.z5.web.core.windows.net/oufr/7.84.1/#/examples/nav

The issue repros on the first example in the Nav controls page. The highlighted "Documents" link has its accessible name overridden with the text "Selected", so a screen reader user would be unaware of what the text of the link actually is.

#### Actual behavior:

The "Documents" link has the accessible name "Selected"

#### Expected behavior:

The "Documents" link should get its accessible name from the text content (i.e. "Documents"), and have `aria-current="page"` set.

### Documentation describing expected behavior
https://w3c.github.io/aria/#aria-current
https://tink.uk/using-the-aria-current-attribute/
